recipes/tdrmapper/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: tdrmapper
+  version: '1.0'
+
+source:
+    git_url: https://github.com/lpantano/tDRmapper
+    git_rev: 06e0b9e
+
+build:
+  number: 1
+
+requirements:
+  run:
+
```

Comment:
Would be good to have a test case here. E.g. try out ``TdrMappingScripts.py --help`` or something like that.